types/mosca/index.d.ts

```diff
@@ -0,0 +1,86 @@
+// Type definitions for mosca 1.0
```

Comment:
Are you sure that these types are correct for mosca 1.0? For example, I can't find a `reason` parameter for `Client.close` function in versions 1.0.x, and in 1.4.x (latest 1.x.x tag on github), but it is present in mosca 2.0.

Reply:
This version is the original library version? I thought it was the version of type definitions file.

Reply:
I don't know if this is enforced in any way, but it's definitely common. For example, look at [this commit](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/392a758ba5456dbe13582a222e839d098535caf5#diff-c907049239fb9f60cedb4ac34fc0d24a) which adds eslint typings - it's added with version 4.16, which seems to match a version number for eslint from around that time (current version is 4.19.1)

Reply:
I'll keep it that way for now, in case another reviewer comments on it I can change.

Reply:
The version number is definitely for the version of the library that the definitions cover, so if this is for mosca 2.0, it should be 2.0. If the definitions are for the newest version of mosca, whatever that happens to be, you can also skip the version number.

Reply:
@sandersn Thanks for your clarification. Fixed.